data/spells/scripts/support/chameleon.lua

```diff
@@ -0,0 +1,16 @@
+local condition = Condition(CONDITION_OUTFIT)
+condition:setTicks(200000)
+
+function onCastSpell(creature, variant, isHotkey)
+	local item = Tile(variant:getPosition()):getTopDownItem()
+	if not item or item.itemid == 0 or not isMoveable(item.uid) then
```

Comment:
I remember far back somewhere in some discussions where Mark stated that we were probably going to get rid of such fields (``item.itemid``, ``item.type``, etc), instead we should be using ``item:getId()`` everywhere xD.

Does ``item:getType():isMoveable()`` achieve the same as ``isMoveable(item.uid)``, if it doesn't you could also replace ``item.uid`` with ``item:getUniqueId()``...